assets/mercury/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: "3.4"
+# we can spin it up
+# so that we add it
+#
+services:
+  db:
+    image: postgres:13.7
+    # Turn off db logs by default
+    # https://postgresqlco.nf/doc/en/param/log_statement/
+    command: ["postgres", "-c", "log_statement=none"]
+    environment:
+      POSTGRES_DB: mercury
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "sh -c 'pg_isready -U postgres -d mercury'"]
+      interval: 5s
+      timeout: 2s
+      retries: 5
+  mq:
+    image: rabbitmq:3.10.20-management
+    hostname: gx-bunny
+    environment:
+      RABBITMQ_DEFAULT_USER: gx_prod
+      RABBITMQ_DEFAULT_PASS: password
+    ports:
+      - 8042:15672
+      - 5672:5672
+  db-provisioner:
+    image: 258143015559.dkr.ecr.us-east-1.amazonaws.com/mercury/provisioner:0.4.1
+    platform: linux/amd64
+    environment:
+      PGPASSWORD: postgres
+      APP_DB_API_USER: postgres
+      APP_DB_API_USER_PASSWORD: postgres
+      APP_DB_ADDR: db
+    # Note: We are overwriting the existing command, just adding a retry
+    #       In Cloud environment, we connect to RDS so no need to worry about starting up
+    #       In local, the DB needs a few seconds to boot up and listen
+    command: 'bash -c "until ./init-db.sh; do sleep 5; done;"'
+    depends_on:
+      db:
+        condition: service_healthy
+  mercury-service-api:
+    image: 258143015559.dkr.ecr.us-east-1.amazonaws.com/mercury/api:latest
```

Comment:
Is this how we make sure we are always using the latest version of mercury?

Reply:
not sure, but I can try to find out :). This and the next question

Reply:
I'm OSS we grab the latest mercury that is on ECR (amazon's container repository). There is mercury build step that pushes the image to ECR.